.github/workflows/build.yml

```
@@ -37,3 +37,30 @@ jobs:
         with:
           reports: |
             reference/**/build/reports/checkstyle/*.xml
+
+# :::automatic-testing:game-test:run-client-game-test
+  client_game_test:
+    runs-on: ubuntu-latest
```

All software in Actions must be locked to a specific version.

```suggestion
    runs-on: ubuntu-24.04
```